src/cmdstan-guide/installation.Rmd

```diff
@@ -8,25 +8,30 @@ will install a pre-built version of CmdStan along with the required
 dependencies (i.e. a C++ compiler, a version of Make, and required
 libraries) detailed below under [Source installation].
 
-One can create a new conda environment and install CmdStan with
-the following command.
-
+One can create a new conda environment (named `stan`)
+and install CmdStan with the following command.
 ```
-conda create -n cmdstan -c conda-forge cmdstan
+conda create -n stan -c conda-forge cmdstan
```

Comment:
[optional]
Would it be useful to remind the user how to check which version of cmdstan is installed?

Reply:
Something like this?

> 
> You can check which version of CmdStan is installed in your current environment by running 
> ```
> conda list cmdstan
> ```
> and you can specify a different version for installation by adding `=VERSION` to the end of the above commands.

Reply:
That's great if you can do it through conda.  I think that'd help people verify they've actually got the version they want.